cmd/dendrite-demo-yggdrasil/main.go

```diff
@@ -140,7 +140,7 @@ func main() {
 		RoomserverAPI:       rsAPI,
 		UserAPI:             userAPI,
 		StateAPI:            stateAPI,
-		KeyAPI:              keyserver.NewInternalAPI(base.Cfg, federation),
+		KeyAPI:              keyserver.NewInternalAPI(base.Cfg, federation, userAPI),
```

Comment:
When you make changes to `cmd/dendrite-demo-yggdrasil`, please also make the same changes to `build/gobind`, as that's what the iOS demo is built from.

Reply:
Can we build it as part of `build.sh`? Else I'll never remember.

Reply:
Unfortunately it requires Xcode and gomobile, and takes ages to build. :-(